api/v1alpha3/cluster_types.go

```diff
@@ -224,6 +225,7 @@ func (c *Cluster) SetConditions(conditions Conditions) {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:deprecatedversion
```

Comment:
Do we actually need the marker on all list types?

Reply:
No -these actually have no effect, I added the markers wherever we had `object:root` - removed in the latest version.